src/main/java/org/carlmontrobotics/Constants.java

```diff
@@ -15,6 +15,14 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+	///array indexed
```

Comment:
explain the indexes

Reply:
these should also be in a subsystem wrapper of Constants, and not just inside constants